client/src/components/menubar/infoMenu.js

```diff
@@ -58,9 +64,16 @@ const InformationMenu = React.memo((props) => {
                 text="Privacy Policy"
               />
             ) : null}
+            {userinfo.is_authenticated ? (
```

Comment:
"Log Out"  should be shown if the authentication type "requires_client_login" and the user "is_authenticated".

The way this is written the "Log Out" menu item would be shown when the server is configured for session based authentication.  In session based (which is the default), the user is automatically logged in, and there is no way to logout, so the Log Out button should not be shown.

Reply:
Nice catch! Thanks, will change